agent/app/agent_capability_windows.go

```diff
@@ -16,10 +16,22 @@
 package app
 
 import (
+	"path/filepath"
+
+	"github.com/aws/amazon-ecs-agent/agent/config"
 	"github.com/aws/amazon-ecs-agent/agent/ecs_client/model/ecs"
 	"github.com/aws/amazon-ecs-agent/agent/taskresource/volume"
 )
 
+var (
+	capabilityDepsRootDir          = filepath.Join(config.AmazonProgramFiles, "managed-agents")
```

Comment:
should this be in `C:\Program Files\Amazon\ECS\managed-agents`? (i.e. Add `ECS` folder)

Reply:
Are there only ECS managed Agents? In that case yes this should be under ECS rather than Amazon.